apps/snitch_core/priv/repo/migrations/20180509123728_add_tax_rate_table.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Snitch.Repo.Migrations.AddTaxRateTable do
+  use Ecto.Migration
+
+  def change do
+    create  table(:snitch_tax_rates) do
+      add :name, :string
+      add :value, :decimal
+      add :calculator, :string
```

Comment:
Please put some NOT NULL constraints here. IMO the name should be unique too.

Reply:
`name` can't be unique as two different zones can have a tax with the same name e.g. GST. It's used in both Europe as well as India.

Reply:
So what's the (compound) primary key for this relation?